realm/realm-library/src/syncIntegrationTest/java/io/realm/objectserver/utils/HttpUtils.java

```diff
@@ -63,6 +66,10 @@ public static void startSyncServer() throws Exception {
         SystemClock.sleep(2000);
```

Comment:
We should remove the sleep, issue above was resolved https://github.com/realm/ros/issues/389

Reply:
Nice! will remove it in another PR